### PR TITLE
DOCS-1333: WAF UI enhancements

### PR DESCRIPTION
**Pull request by girishshankaran**
Product Version(s):

Issue:
(https://tigera.atlassian.net/browse/DOCS-1333)

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 1]
```diff
@@ -74,14 +74,18 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
...
-    src='/img/calico-enterprise/waf-services.png'
+2. On the Select Services to Inspect page, choose all the services for which you want WAF to inspect the traffic.
+    
+   <img
+    src='/img/calico-enterprise/waf-select-services.png'
     alt='WAF services'
     width='600'
   />
 
+:::note
```
Comment (LorcanMcVeigh) [inline calico-enterprise/threat/web-application-firewall.mdx]: I'd add this as step 3. and add an image of the 3rd page maybe ?

Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: @LorcanMcVeigh it's done

**Review by LorcanMcVeigh — APPROVED** [review]

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 2]
```diff
@@ -74,8 +74,16 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
 ### Enable WAF
 
 1. On the Manager UI, click **Threat Defense**, **Web Application Firewall**, **Configure Web Application Firewall**.
-2. For each service that you want to enable for WAF, click the **Action** column, select Enable.
+2. On the Select Services to Inspect page, choose all the services for which you want WAF to inspect the traffic.
```
Comment (ctauchen) [inline calico-enterprise/threat/web-application-firewall.mdx]: 2. Select the services you want WAF to inspect, and then click **Confirm Selections**.
3. On the **Web Application Firewall** page, you can verify that WAF is enabled for a service by locating the service and checking that the **Status** column says **Enabled**.
4. To make further changes to a service, click **Actions**, and then **Enable** or **Disable**.

**Review by ctauchen — COMMENTED** [review]
A few suggestions for the step procedure. My main aim here is to make everything hang together from the same starting point: a user who hasn't yet set up WAF. 
In the previous version, step 3 creates a new starting point (you're a user who has already set up WAF).

I also think we should kill the image. We can consider returning it later.

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 2 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: It's fixed

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 3]
```diff
@@ -74,14 +74,17 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
...
-    src='/img/calico-enterprise/waf-services.png'
+2. Select the services you want WAF to inspect, and then click **Confirm Selections**.
+    
+   <img
+    src='/img/calico-enterprise/waf-select-services.png'
     alt='WAF services'
     width='600'
   />
 
+3. On the Web Application Firewall page, you can verify that WAF is enabled for a service by locating the service and checking that the **Status** column says **Enabled**.
```
Comment (ctauchen) [inline calico-enterprise/threat/web-application-firewall.mdx]:
```suggestion
3. On the **Web Application Firewall** page, you can verify that WAF is enabled for a service by locating the service and checking that the **Status** column says **Enabled**.
```

Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Fixed

**Review by LorcanMcVeigh — APPROVED** [review]

**Pull request merged**